draft-wing-cidfi.md

```diff
@@ -249,7 +255,18 @@ client and server and with the participating CIDFI network elements.
 
 
 
-# Network Preparation: DNS SVCB Records
+# Network Preparation
+
+The network is configured to advertise its support for CIDFI.
+
+For this step, four mechanisms are described in this document: DNS
+SVCB records, IPv6 Provisioning Domains (PvD), DHCP, and 3GPP PCO.
+These are described below.
+
+> Standardizing on all or some of these mechanisms is for further discussion.
```

Comment:
I don't think standardizing 3GPP PCO is in the scope of IETF. Why do we have to discuss about "3GPP PCO" ?

Reply:
Agree we don't need to specify the PCO IE in this draft as this is a 3GPP thing. 

That's said, I think this is covered by "or some of these" in the text proposed by Dan. I think the current wording is OK here.